.github/workflows/release.yml

```diff
@@ -1,11 +1,15 @@
 name: Draft Release
 
 on:
+  push:
+    branches:
+      - main
   workflow_dispatch:
     inputs:
       tag:
         description: 'Version Tag'
-        required: true        
+        required: true
+        default: '0.1.1'        
```

Comment:
Is this meant to have a default?

Reply:
I'm not sure if it makes sense if we have multiple versions of the dev container zip file, so I was thinking maybe we should just use a constant value? Perhaps `latest` is a better choice than `0.1.1`